src/styles/home.css

```diff
@@ -60,3 +60,13 @@
   animation-duration: 5s;
   animation-iteration-count: infinite;
 }
+
+.card .like {
+  cursor: pointer;
+  transition: all ease-in 0.05s;
+  margin-left: 1rem;
+}
+
+.card .like:hover {
+  scale: 1.2;
+}
```

Comment:
- [optional] our requirement is complete, great working there but there is room for improvement we can add animation
or color pink or red for the heart until our like is displayed there 
- [ ] right now our like just in black and some of user click more than one time without being notify there likes are already counted